docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.6"
+
+services:
+  fcrepo:
+    image: fcrepo/fcrepo:latest
+    ports:
+      - "8080:8080"
+      - "8181:8181"
```

Comment:
Hi @mikejritter What's port 8181 used for?

Reply:
I'm not sure -- @dbernstein had it in the  instructions for bringing up the fcrepo container but I don't see any reference to it in the codebase.